ush/wave_grid_interp_sbs.sh

```diff
@@ -45,12 +45,12 @@ for ID in ${waveGRD} ${grdID}; do
 done
 
 # Check if there is an interpolation weights file available, and copy it if so
-if [[ -f "${FIXgfs}/wave/ww3_gint.WHTGRIDINT.bin.${grdID}" ]]; then
-  echo "INFO: Interpolation weights found at: '${FIXgfs}/wave/ww3_gint.WHTGRIDINT.bin.${grdID}'"
-  cp "${FIXgfs}/wave/ww3_gint.WHTGRIDINT.bin.${grdID}" "./WHTGRIDINT.bin"
+if [[ -f "${FIXgfs}/wave/ww3_gint.WHTGRIDINT.bin.${waveGRD}.${grdID}" ]]; then
+  echo "INFO: Interpolation weights found at: '${FIXgfs}/wave/ww3_gint.WHTGRIDINT.bin.${waveGRD}.${grdID}'"
+  cp "${FIXgfs}/wave/ww3_gint.WHTGRIDINT.bin.${waveGRD}.${grdID}" "./WHTGRIDINT.bin"
```

Comment:
While we're here and I see it, please update the `cp` here to `cpreq`:
```suggestion
  cpreq "${FIXgfs}/wave/ww3_gint.WHTGRIDINT.bin.${waveGRD}.${grdID}" "./WHTGRIDINT.bin"
```